web/src/routes/TagDetails/ModelCard/ModelCard.tsx

```diff
@@ -0,0 +1,68 @@
+import {useEffect, useState} from 'react';
+import {Card, PageSection, Divider, TextContent} from '@patternfly/react-core';
+import {useLocation, useSearchParams} from 'react-router-dom';
+
+import { Remark, useRemark } from 'react-remark';
+import remarkGemoji from 'remark-gemoji';
+
+import {
+  ManifestByDigestResponse,
+  Tag,
+  TagsResponse,
+  getManifestByDigest,
+  getTags,
+} from 'src/resources/TagResource';
+import {
+  parseOrgNameFromUrl,
+  parseRepoNameFromUrl,
+  parseTagNameFromUrl,
+} from '../../../libs/utils';
+
+
+export function ModelCard() {
+  const [searchParams] = useSearchParams()
+  const location = useLocation();
+  
+  const org = parseOrgNameFromUrl(location.pathname);
+  const repo = parseRepoNameFromUrl(location.pathname);
+  const tag = parseTagNameFromUrl(location.pathname);
+
+  const [modelcardMarkdown, setModelcardMarkdown] = useState<string>('');
+
+  useEffect(() => {
+    (async () => {
+
+      try {
+	const resp: TagsResponse = await getTags(org, repo, 1, 100, tag);
```

Comment:
We use react query to fetch data. This should be in a hook similar to [this](https://github.com/quay/quay/blob/master/web/src/hooks/UseTags.ts#L11-L33)